modules/35_transport/complex/datainput.gms

```diff
@@ -29,6 +31,25 @@ p35_pass_nonLDV_ES_efficiency(ttot,regi)$(ttot.val ge 2005)  = f35_transp_eff(re
 p35_passLDV_ES_efficiency(ttot,regi)$(ttot.val ge 2005)      = f35_transp_eff(regi,"Eff_Pass_LDV");
 p35_freight_ES_efficiency(ttot,regi)$(ttot.val ge 2005)      = f35_transp_eff(regi,"Eff_Freight");
 
+
+p35_valconv = smin((regi),f35_transp_eff(regi,"Eff_Pass_nonLDV"));
```

Comment:
I'm not quite sure if it is a good Idea to have efficiencies converge to the least efficient public transport system